roles/remove-node/post-remove/tasks/main.yml

```diff
@@ -1,9 +1,54 @@
 ---
+- name: Lookup node IP in kubernetes
```

Comment:
can we lookup node ip in facts? what's stopping us from that?

Reply:
The node is being decommissioned and I don't want to rely on facts. If there is no ip set and no access_ip set, it's going to be in k8s at least. I think checking if the fact hostvars[node]['ansible_default_ipv4']['address'] is set is kind of pointless